ftw/iframeblock/browser/templates/iframeblock.pt

```diff
@@ -9,6 +9,15 @@
                             scrolling python:'no' if context.auto_size else 'auto'"
             width="100%"
             class="iframeblock loading" onload="onIframeLoaded(this)"></iframe>
-    <script tal:condition="context/auto_size">iFrameResize()</script>
+    <script tal:condition="context/auto_size">
+      iFrameResize({
+        inPageLinks: true,
+        resizedCallback: function () {
+          // Always scroll a bit above iframe after a new page is loaded
+          var offset = $('.sl-layout').offset().top
```

Comment:
Why not scroll to the top?
When auto-size is active, the user should not notice that there is an iframe.

Reply:
After testing it for a bit, it felt more natural to scroll a bit above the content instead of to the very top. If you scroll to the top every time, the user will most likely have to scroll back down manually to see every bit of content.